node-notes-cli/app.js

```diff
@@ -0,0 +1,46 @@
+import { readFile, writeFile } from 'node:fs/promises';
+
+async function parse() {
+  const file = await readFile('data.json');
+  const parse = await JSON.parse(file);
+  return parse;
+}
+
+const data = await parse();
+
+async function notes() {
+  const list = data.notes;
+  for (const [key, value] of Object.entries(list)) {
+    console.log(`${key}: ${value}`);
+  }
+}
+async function create(newNotes) {
+  const key = data.nextId;
+  data.notes[key] = newNotes;
+  data.nextId++;
+  const newObj = JSON.stringify(data, null, 2);
+  await writeFile('data.json', newObj);
+}
+
+async function remove(index) {
+  delete data.notes[index];
```

Comment:
It would be nice to tell the user if the note doesn't exist. Otherwise they might not understand why the command didn't seem to do anything.